src/result.jl

```diff
@@ -8,10 +8,10 @@ the population, the number of iterations and the number of function calls.
 struct Result
     fxstar  # optimum
     xstar  # optimizer
-    population  # population at last state
-    n_iters # pass the number of iterations
-    n_evals  # number of individual evaluations
-    runtime
+    population::Int64  # population at last state
```

Comment:
```suggestion
    population::AbstractVector  # population at last state
```
Vector of vectors, or vector of Particles; we don't know the type nor dimension of the things inside but that's ok